test/system/helpers.bash

```diff
@@ -341,6 +341,13 @@ function is_cgroupsv2() {
     test "$cgroup_type" = "cgroup2fs"
 }
 
+# True if podman is using netavark
+function is_netavark() {
+    # FIXME: Do the tests ever run against other versions?
+    # Can't safely use --filter: < v4.0 do not have the networkBackend field.
+    $PODMAN info | grep -qiw 'networkBackend:.*netavark'
```

Comment:
Why do you not use `podman info --format {{.Host.NetworkBackend}}`. The test always have to match the podman version. There is no point in running the current system tests with an older version as they will obviously fail because new tests have been added.

Reply:
Okay, right.  Ya I wasn't sure about that.  Can certainly use `--format` then.

Reply:
Okay, this is in place now.